### PR TITLE
Update the readme to point out the new F/T sensor and its guide

### PR DESCRIPTION
**Pull request by Samahu**
## Linked Issues:
| EPIC ⚡| [OCEANWATER-614 / Implement a force torque sensor for the lander arm ](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-614) |
| :----------- | :----------- |
| Jira Ticket 🎟️   | [OCEANWATER-619 / Update the user guide to include utilization of the FT Sensor ](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-619) |
| Github :octocat:  | N/A |

## Summary of Changes
* Updated the ow_lander/README.md
* A [new section for the F/T sensor within the wiki](https://github.com/nasa/ow_simulator/wiki/Lander-Simulation#force-torque-sensor) 

## Test
* N/A

## Objectives
* Review the new user guide
* Any revisions needed?

### CODE REVIEW

File: ow_lander/README.md [hunk 1]
```diff
@@ -136,7 +136,11 @@ the number of joints or any joint names, this header must be updated as well.
 
 ### Package History
 
-A major overhaul of this package was done after switching from JointPositionController
+* A major overhaul of this package was done after switching from JointPositionController
 to JointTrajectoryController for arm services. Nonetheless, this revised version
 is more comptabile with the file generated by the setup_assistant with few manual
 edits. Refer to PR: [Oceanwater 518 switch to using effort controllers joint trajectory controller](https://github.com/nasa/ow_simulator/pull/40) for complete track of the set of changes.
+
+* An F/T sensor was added to the arm on the distal pitch joint. For more information
```
Comment (mogumbo) [inline ow_lander/README.md:144]: I recommend "force/torque" instead of "F/T" for clarity.

**Review by mogumbo — APPROVED** [review]

File: ow_lander/README.md (hunk 1 above)
Comment (Samahu) [reply ow_lander/README.md]: addressed

**Comment by kmdalal** [comment]
I suggest adding a comment in the ow_simulator wiki that this is a Release 8 feature, since it's not in the master branch.  An alternate approach is putting the new documentation on a branch of the wiki, and merging this upon release.   Not a huge deal here since we are (hopefully) close to release 8.

**Review by kmdalal — APPROVED** [review]

**Comment by Samahu** [comment]
> I suggest adding a comment in the ow_simulator wiki that this is a Release 8 feature, since it's not in the master branch. An alternate approach is putting the new documentation on a branch of the wiki, and merging this upon release. Not a huge deal here since we are (hopefully) close to release 8.

I would have have delayed merging my wiki changes to the master branch if it didn't seem like we keep updating the `master` branch on the go. You can see clearly that I had my wiki changes initially on a branch  and then merged them into master because it already contains doc for unreleased features.

**Pull request merged**